src/test/suite/fixtures/recorded/selectionTypes/changeEveryPaintPairParen2.yml

```diff
@@ -0,0 +1,36 @@
+languageId: plaintext
+command:
+  spokenForm: change every paint pair paren
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: (}
+      modifiers:
+        - type: everyScope
+          scopeType: {type: nonWhitespaceSequence}
+        - type: containingScope
+          scopeType: {type: surroundingPair, delimiter: any}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: foo(aa|aa-().c_d bb|bb-().'') { return 1; }
+  selections:
+    - anchor: {line: 0, character: 1}
+      active: {line: 0, character: 1}
+  marks:
+    default.(:
+      start: {line: 0, character: 3}
+      end: {line: 0, character: 4}
+finalState:
+  documentContents: "  { return 1; }"
```

Comment:
It may be unintuitive for some users that the first paint extends outside the selected range. Not sure if that's something we'd want to change though since it is one contiguous nonwhitespace sequence.

Reply:
This looks like the correct implementation with regards to what we talked about